lib/coherence/schema.ex

```diff
@@ -151,7 +151,7 @@ defmodule Coherence.Schema do
         Returns a changeset ready for Repo.update
         """
         def unlock(user) do
-          Config.user_schema.changeset(user, %{locked_at: nil, unlock_token: nil, failed_attempts: 0})
+          Config.user_schema.changeset(user, %{locked_at: nil, unlock_token: nil, failed_attempts: 0, current_password: user.password})
```

Comment:
The current_password should not be required here. Please update the changeset logic so it only requires current_password validation if the password is being changed.